.buildkite/pipeline.bb.yml

```diff
@@ -0,0 +1,52 @@
+steps:
+
+  - label: 'BitBar Poc - iOS 15 - batch 1'
+    depends_on: "cocoa_fixture"
+    timeout_in_minutes: 90
+    agents:
+      queue: opensource
+    plugins:
+      artifacts#v1.9.0:
+        download: "features/fixtures/ios/output/iOSTestApp.ipa"
```

Comment:
Should this be `ipa_url.txt` as well?

Reply:
Eventually, yes, but we're only uploading the app to BrowserStack at the moment.  We can do the same for BitBar in time, but it's more effort than I wanted to put in at this stage.